layouts/default.hbs

```diff
@@ -28,7 +28,9 @@
 
 		{{> _global-header }}
 
-		{{ body }}
+		<main id="main" class="main">
```

Comment:
I'm not sure how much I like main in the default layout. I usually prefer to put these in the actual template, because there are occasionally things in the template that don't need to go in the main section, like mastheads or special areas above the footer.

Reply:
I put it here because I've found this is something we often forget. Would it make more sense to create a level-specific template and add a main tag around the body content there?

Reply:
It's hard to say. It tends to vary per project. I might prefer having the main tag be built into the boilerplate template.hbs file from the generator itself than in a layout, if forgetting to add it is the issue. Then it can just be removed if needed.

Reply:
I can move it. Where is template.hbs? I can't find it in here.

Reply:
I've updated this in the main generator repo.